### PR TITLE
docs: fix typo

### PR DESCRIPTION
**Pull request by RajendraUppu**
### ❓ Type of change

- [x] 📖 Documentation (updates to the documentation or readme)
- [ ] 🐞 Bug fix (a non-breaking change that fixes an issue)
- [ ] 👌 Enhancement (improving an existing functionality like performance)
- [ ] ✨ New feature (a non-breaking change that adds functionality)
- [ ] ⚠️ Breaking change (fix or feature that would cause existing functionality to change)

### 📝 Checklist

- [ ] I have linked an issue or discussion.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

File: docs/content/1.getting-started/4.assets.md [hunk 1]
```diff
@@ -75,7 +75,7 @@ export default defineNuxtConfig({
     css: {
       preprocessorOptions: {
         scss: {
-          additionalData: '@use "@/assets/_colors.scss" as *;'
+          additionalData: '@use "@/assets/_colors.scss" as *';
```
Comment (danielroe) [inline docs/content/1.getting-started/4.assets.md:78]: I don't think this is correct.

File: docs/content/1.getting-started/5.seo-meta.md [hunk 2]
```diff
@@ -21,6 +21,16 @@ useHead({
...
+  link: [
+      {
+        href: "https://third-party-css.com/main.css",
+      },
+    ],
+  script: [
+    {
+      src: 'https://third-party-script.com/script.js',
+    }
+  ],
```
Comment (danielroe) [inline docs/content/1.getting-started/5.seo-meta.md:33]: We're reworking this section in https://github.com/nuxt/framework/pull/8000.

**Review by danielroe — CHANGES_REQUESTED** [review]
Would you update your branch to be a pull request to `main` instead, and remove the seo-management update? (The change was good, but we are reworking this in https://github.com/nuxt/framework/pull/8000 instead.)

File: docs/content/1.getting-started/4.assets.md (hunk 1 above)
Comment (RajendraUppu) [reply docs/content/1.getting-started/4.assets.md:78]: Yes! I was about to remove that ; instead I have added after My bad Sorry 🙁

**Pull request closed without merging**